.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,6 @@
+{
+  "ExpandedNodes": [
+    ""
+  ],
+  "PreviewInSolutionExplorer": false
```

Comment:
@edgarEOS 

These are all files that you would never check in, since they are generated from a build or contain machine-specific information.

Let's add the `.vs` folder to the `.gitignore` file